pkg/config/tlscfg/options.go

```diff
@@ -52,12 +53,18 @@ func (p *Options) Config(logger *zap.Logger) (*tls.Config, error) {
 		return nil, fmt.Errorf("failed to get cipher suite ids from cipher suite names: %w", err)
 	}
 
+	minVersionId, err := VersionNameToID(p.MinVersion)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get minimum tls version: %w", err)
+	}
+
```

Comment:
```suggestion
	var minVersionId uint16
	var err error
	if p.MinVersion != "" {
	    minVersionId, err = VersionNameToID(p.MinVersion)
	    if err != nil {
		    return nil, fmt.Errorf("failed to get minimum tls version: %w", err)
	    }
	}
```
i.e. do not change the default in Go struct if user did not provide any value